web/app/styles/app.css

```diff
@@ -47,8 +47,8 @@
 }
 
 .borderless td, .borderless th {
-  border: none;
-  border-top: 0px;
+  border: none !important;
```

Comment:
border: none is equivalent to border:0 
Refer: https://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0 
so, don't know why below statement is required.